src/mscorlib/src/System/RtType.cs

```diff
@@ -618,7 +618,11 @@ private unsafe RuntimeMethodInfo[] PopulateMethods(Filter filter)
                         while (RuntimeTypeHandle.IsGenericVariable(declaringType))
                             declaringType = declaringType.GetBaseType();
 
-                        bool* overrides = stackalloc bool[RuntimeTypeHandle.GetNumVirtuals(declaringType)];
+                        int numVirtuals = RuntimeTypeHandle.GetNumVirtuals(declaringType);
+
+                        bool* overrides = stackalloc bool[numVirtuals];
+                        new Span<bool>(overrides, numVirtuals).Clear();
```

Comment:
I'd previously tried running all of the corefx tests with that hack to disable init locals, and I'd seen maybe three or four tests fail total... I assume this is the fix for those?

Reply:
Yes, this was a few places in CoreLib that used to depend on stackalloced memory to be initialized. I have just found a second one in TraceLoggingEventSource.cs.